src/ui/branding-info-ui.svelte

```diff
@@ -14,21 +15,21 @@
         srcWebp={buildAssetURL(brandingInfo.app_icon_webp)}
       />
     {/if}
-    <span class="app-title">{brandingInfo.seller_company_name}</span>
+    <span class="app-title"><Localized>{brandingInfo.seller_company_name}</Localized></span>
```

Comment:
this is complaining that it's missing props

Reply:
Shouldn't be complaining anymore, all props are optional now.